sdk/README.md

```diff
@@ -1,25 +1,29 @@
 # Lookout Analyzer SDK
 
-An analyzer is a gRPC server, that implements [Analyzer service](https://github.com/src-d/lookout/tree/master/sdk/service_analyzer.proto) to receive events from the server.
+An analyzer is a [gRPC](https://grpc.io/) server that implements the [protocol buffers Analyzer service](./service_analyzer.proto) to receive events from the lookout server.
 
-An analyzer should use gRPC client to access [Data service](https://github.com/src-d/lookout/tree/master/sdk/service_data.proto) to get content or UAST of the changes.
+At the same time, an analyzer will act as a gRPC client to access the [Data Server](./service_data.proto) exposed by the lookout server. This endpoint will provide the necessary source code and [Babelfish Universal Abstract Syntax Trees (UAST)](https://doc.bblf.sh/uast/uast-specification.html).
 
-All `.proto` files are located in [/sdk](https://github.com/src-d/lookout/tree/master/sdk) directory.
+All the needed `.proto` files are located in the current [/sdk directory](./).
 
 
 ## Code generation
 
+You can create a new analyzer in any language that supports protocol buffers.
```

Comment:
We should say a few words why we need code generation at all.

gRPC communication is performed in protocol buffers format, and it is possible to generate the source code to send and receive messages. Blah-blah-blah